samples/sqltransport-sqlpersistence/SqlServer_3/Receiver/OrderLifecycleSaga.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+using NServiceBus.Persistence.Sql;
+
+[SqlSaga(
+    correlationProperty: nameof(OrderLifecycleSagaData.OrderId)
+ )]
+public class OrderLifecycleSaga :
+    Saga<OrderLifecycleSagaData>,
```

Comment:
Trying to limit new concepts, since SqlSaga is optional. I should probably mention the attribute in the sample text though.